source/Cast.h

```diff
@@ -0,0 +1,113 @@
+/* Cast.h
+Copyright (c) 2014 by Michael Zahniser
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef CAST_H_
+#define CAST_H_
+
+#include <cassert>
```

Comment:
Casting should be a rare operation. If we need this class then we are casting too much.

Reply:
I don't actually call the function that asserts all that often (maybe like 5 overall). But as stated below, I'll remove them :)